README.md

```diff
@@ -199,10 +199,18 @@ source: 192.0.2.0/24
 ### interface
 
 String or list of interface name strings.
+Also accepts PCI id's, which is two pairs of hexidecminal numbers (XXXX:XXXX)
 
-```
+If a PCI id corresponds to more than one interface name,
+all interfaces with the PCI id specified will have the play applied.
+
+```yaml
 interface: eth2
 ```
+```yaml
+# PCI id for Intel Corporation Ethernet Connection
+interface: 8086:15d7
```

Comment:
FYI, in the network role, we use a brand new key name (`path`) to specify the PCI address [1], not sure if the firewall role should also have a good key name distinction between the actual interface name and PCI ID.

[1] https://github.com/linux-system-roles/network/blob/main/module_utils/network_lsr/argument_validator.py#L1755

Reply:
Also adding an complete vivid example like the following would be nice:

Allow interface with PCI ID 8086:15d7  in trusted zone:

```yaml
firewall:
  - interface: 8086:15d7
    zone: trusted
    state: enabled
```

Reply:
After thinking about it, I agree on both comments on this conversation, it would be difficult to manage an interface named like a PCI device ID without making the change you suggested or a similar change, although I have no idea why someone would knowingly add an interface with the name in the structure of a PCI id.

Additionally, I will go ahead and add an example to interface using a PCI device ID after making the first suggested change